Source/WebCore/loader/SubframeLoader.cpp

```diff
@@ -328,6 +329,15 @@ RefPtr<LocalFrame> FrameLoader::SubframeLoader::loadSubframe(HTMLFrameOwnerEleme
 
     frame->protectedLoader()->loadURLIntoChildFrame(url, referrerToUse, subFrame.get());
 
+#if ENABLE(CONTENT_EXTENSIONS)
+    if (url.isAboutBlank() && subFrame->page()) {
```

Comment:
I am surprised this works with your test, since `srcdoc` uses `about:srcdoc` as the URL. Should this also check `url.isAboutSrcDoc()`?

Reply:
I'm happy to add `url.isAboutSrcDoc()` there, but I did verify that the test fails without my change in `SubframeLoader` and passes with it.

Reply:
Update: The test was passing with and without the change, but I will keep it around for posterity and to make sure we don't regress this behavior somehow.